x-pack/plugins/ml/public/application/jobs/jobs_list/components/utils.js

```diff
@@ -36,6 +36,23 @@ export function loadFullJob(jobId) {
   });
 }
 
+export function loadClonableJob(jobId) {
```

Comment:
Did you mean to include these changes? Looks like they are part of the `exclude_generated` work.

Reply:
My bad, this commit https://github.com/elastic/kibana/pull/88880/commits/3a50b3cdcccb58d81d86df89f1f70913c3a94a12 should and will be reverted. Thanks for catching that.

Reply:
This is now reverted https://github.com/elastic/kibana/pull/88880/commits/311892828ba8ad9bf92efe643c97c43d7c0439a1